Helper/Session.php

```diff
@@ -167,4 +175,26 @@ public function loadSession($quote)
         }
         $this->replaceQuote($quote);
     }
+
+    /**
+     * Load and set cached form key
+     *
+     * @param Quote $quote
+     */
+    public function setFormKey($quote)
```

Comment:
looks like unused?

Reply:
It is used in /Model/Api/CreateOrder.php